native/SampleApps/SmartSyncExplorer/SmartSyncExplorer/Classes/ContactListViewController.m

```diff
@@ -301,12 +301,15 @@ - (UIView *)accessoryViewForContact:(ContactSObjectData *)contact {
 
     if ([self.dataMgr dataHasLocalChanges:contact]) {
         UIImage *localImage;
-        if ([self.dataMgr dataLocallyCreated:contact])
```

Comment:
Show deleted image if deleted (regardless of whether it was created or updated also)
Show created image if created but not deleted (regardless of whether it was also updated)